File: component/src/main/java/com/mackwu/component/ui/widget/DialogActivity.java
package com.mackwu.component.ui.widget;

import android.os.Bundle;

import androidx.annotation.Nullable;

import com.mackwu.base.BaseActivity;
import com.mackwu.base.viewmodel.BaseViewModel;
import com.mackwu.component.databinding.WidgetActivityDialogBinding;
import com.mackwu.component.ui.dialog.LoadingDialog;

/**
 * ===================================================
 * Created by MackWu on 2021/4/22 15:15
 * <a href="mailto:wumengjiao828@163.com">Contact me</a>
 * <a href="https://github.com/mackwu828">Follow me</a>
 * ===================================================
 */
public class DialogActivity extends BaseActivity<BaseViewModel, WidgetActivityDialogBinding> {

    @Override
    public void initView(@Nullable Bundle savedInstanceState) {
        LoadingDialog loadingDialog = new LoadingDialog();
        binding.btnShow.setOnClickListener(v -> loadingDialog.show(this));
        binding.btnHide.setOnClickListener(v -> loadingDialog.dismissAllowingStateLoss());
    }

}

File: view/src/main/java/com/mackwu/view/basic/DialogActivity.kt
package com.mackwu.view.basic

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.mackwu.view.R
import kotlinx.android.synthetic.main.activity_dialog.*

/**
 * ===================================================
 * Created by MackWu on 2020/1/20 15:25
 * <a href="mailto:wumengjiao828@163.com">Contact me</a>
 * <a href="https://github.com/mackwu828">Follow me</a>
 * ===================================================
 */
class DialogActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_dialog)

        btn_dialog.setOnClickListener {
        }
    }


}